steps/examples/step-base/lib/angular/template/base-form.tpl.html

```diff
@@ -0,0 +1,26 @@
+<p class="wfm-accident-row">Please register details</p>
```

Comment:
Clean that up so it will not be accident related. Just single item. Let's say `name`